samples/modules/lvgl/demos/boards/mimxrt595_evk_mimxrt595s_cm33.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright 2023, NXP
```

Comment:
Copyright should be updated- also, won't the display running with this sample on the RT595 use RGB565, not ARGB8888?